Cargo.toml

```diff
@@ -31,6 +31,7 @@ serde_json = "1.0.66"
 quickcheck = { version = "1.0.3", optional = true }
 derive_builder = { version = "0.10.2", optional = true }
 getset = { version = "0.1.1", optional = true }
+path-clean = "0.1.0"
```

Comment:
@saschagrunert What do you think about the introduction of this crate?
https://github.com/danreeves/path-clean

Reply:
Would it make sense to make it optional and make rootless a feature?

Reply:
> Would it make sense to make it optional and make rootless a feature?

I think it is OK.